examples/forwarding-refs/fancy-button-ref.js

```diff
@@ -3,13 +3,13 @@ import FancyButton from './FancyButton';
 // highlight-next-line
 const ref = React.createRef();
 
-// The FancyButton component we imported is the LogProps HOC.
-// Even though the rendered output will be the same,
-// Our ref will point to LogProps instead of the inner FancyButton component!
-// This means we can't call e.g. ref.current.focus()
+// Le composant FancyButton que nous avons importé est le HOC LogProps.
+// Même si le rendu sera le même,
```

Comment:
```suggestion
// Même si l’affichage sera le même,
```

Il ne faut pas avoir peur de recaler le *wrapping* si besoin ; je le ferai dans ma repasse visuelle finale